platform/features/plot/bundle.js

```diff
@@ -84,12 +88,30 @@ define([
                     ]
                 }
             ],
+            "services": [
+                {
+                    "key": "ExportImageService",
```

Comment:
A nit, but `exportImageService` would be more consistent with naming conventions used elsewhere. (Same goes for its usage in PlotController; we use UpperCamelCase as a convention to identify constructor functions.)